### PR TITLE
New design for the maintenance mode

### LINKED ISSUE #7620: New design for Maintenance mode
Feedback from @quba - the maintenance mode is not very usable. 

 * Let's review the design (maybe we ask designers)
 * review the copy to make it non-technical and user friendly

Current design:

capture d ecran 2015-04-06 a 10 37 57

### PR DESCRIPTION
**Pull request by mnapoli**
Fixes #7620 

Pull request based on #7578, don't merge this one before merging #7578.

Current design:

There's not much to say in that page so it feels a bit empty…

### CODE REVIEW

**Comment by mattab** [comment]
the visual looks good to me, a solid improvement!
 
(I didn't look at code)

**Comment by mnapoli** [comment]
FYI the diff is huge because this PR is based on the other PR which is also based on another PR :)

But the commit is very simple: 71c7a7d4d2f29634c621d5aaedaa8eee09880a01

File: plugins/CoreUpdater/templates/layout.twig [hunk 1]
```diff
@@ -42,13 +42,17 @@
...
 <!--[if (gte IE 9)|!(IE)]><!-->
 <body id="simple" ng-app="app"><!--<![endif]-->
-<div id="contentsimple">
-    <div id="title">
-        <img title='Piwik' alt="Piwik" src="plugins/Morpheus/images/logo-header.png" style="margin-left:10px;"/>
-        <span id="subh1"> # {{ 'General_OpenSourceWebAnalytics'|translate }}</span>
-    </div>
+
+<div class="logo">
+    <img title="Piwik" alt="Piwik" src="plugins/Morpheus/images/logo-header.png"/>
```
Comment (tsteur) [inline plugins/CoreUpdater/templates/layout.twig]: Should we show a possible custom logo here?

Comment (mnapoli) [inline plugins/CoreUpdater/templates/layout.twig]: This is the diff for #7578 (updater redesign).

I've added support for custom logo in the maintenance mode, I'll try to add support for that in the updater in #7578.

Comment (mnapoli) [inline plugins/CoreUpdater/templates/layout.twig]: Added support for custom logos in the updater too (in #7578).

File: core/FrontController.php [hunk 2]
```diff
@@ -412,28 +413,27 @@ protected function prepareDispatch($module, $action, $parameters)
...
-            $response = new ResponseBuilder($format);
-            echo $response->getResponseException($exception);
-            exit;
+        $logoUrl = null;
+        $faviconUrl = null;
+        try {
+            $logo = new CustomLogo();
+            $logoUrl = $logo->getHeaderLogoUrl();
+            $faviconUrl = $logo->getPathUserFavicon();
+        } catch (Exception $ex) {
```
Comment (mattab) [inline core/FrontController.php]: +1 - I tested that maintenance is displayed correctly when DB is not accessible and it works thanks to this catch (case where the DB could be down during maintenance)

**Comment by mnapoli** [comment]
Keep in mind while reviewing that this PR is based on #7578, only the 2 last commits are about the maintenance.

**Comment by mattab** [comment]
* the subtitle we can use Piwik::getRandomTitle() instead of `free/libre analytics platform`
 * otherwise looks good +1 to merge

**Comment by mnapoli** [comment]
It now uses `Piwik::getRandomTitle()`

**Comment by mnapoli** [comment]
@mattab I'm afraid the commits are not good here :/ I was rebasing this PR because as you can see the diff includes some commits of the updater redesign, which is not good. I don't know if there are any consequences

**Pull request merged**